cove_ocds/views.py

```diff
@@ -15,6 +16,7 @@
 from cove.lib.exceptions import CoveInputDataError, cove_web_input_error
 from cove.views import explore_data_context
 
+from cove_ocds.webservice import views
```

Comment:
This module doesn't exist.

Reply:
The module has been taken out. It was a remnant that slipped through from the previous version. Please how can I check up the Travis CI build errors?

Reply:
>  Please how can I check up the Travis CI build errors?
You can check Travis build errors by clicking on the red X in the latest commit at which the build failed. You will be redirected to travis-ci.org/OpenDataServices..etc, where you will find two builds (Python 3.4 and 3.5). Clicking on the builds will show you the logs with the errors.

Reply:
Thanks @edugomez. I've had a look but perhaps a summary of the errors would be more useful at this point.

Reply:
@patxiworks you can ignore this error ```test_url_input[tenders_releases_1_release_with_extensions_new_layout.json-expected_text2-not_expected_text2-True]```, it is not due to your code (it will be fixed when PR #852 gets merged -- this commit specifically https://github.com/OpenDataServices/cove/pull/852/commits/5dc875ca54fe4fd50abe36a1c8c021872a58dcc0)